middleware/middleware-1.3.23.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: /release-notes/middleware/1.3.23
+title: Version 1.3.23
+---
+
+# fiskaltrust.Middleware 1.3.23 (Germany)
+_September 15, 2021_
+
+In this version of the Middleware, we added the possibility to easily identify mandatory and optional signature items when printing receipts. Additionally, the Middleware is now able to automatically update the _Fiskal Cloud Connector_, and several bugs and stability issues have been resolved.
+
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.
+
+:::
+
+## Feature: Flag optional signature items
+Starting with this version, we're adding the _ftSignatureFormatFlag_ `0x0000000000010000` to signatures that are not mandatory to print. Especially with the recently changed _KassenSichV_ regulations in Germany (which allow to omit printing most texts when a QR code is added), this should enable PosCreators to more easily decide which information needs to be printed onto the receipt and which information is optional.
+
+More information can be found in the [respective docs](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/germany/reference-tables/ftsignatureformat). Please note that this behavior can be deactivated by setting the Queue configuration parameter `FlagOptionalSignatures` to `false`.
+
+## Feature: Automatically update FCC with SCU update
+Starting from this version, each _SwissbitCloud_ and _DeutscheFiskal_ SCU are connected to a specific minimal Fiskal Cloud Connector (FCC) version. If the currently installed FCC version is lower than this, the Middleware will automatically download and update the newer FCC. This should enable users that have already installed previous versions to profit from the latest improvements in this external dependency.
```

Comment:
```suggestion
Starting from this version, each _SwissbitCloud_ and _DeutscheFiskal_ SCU is tied to a specific minimal Fiskal Cloud Connector (FCC) version. If the currently installed FCC version is lower than this, the Middleware will automatically download and update the newer FCC. This should enable users that have already installed previous versions to profit from the latest improvements in this external dependency.
```

Reply:
PR was already merged, but I fixed it in `master` - thanks!